elastalert/alerts.py

```diff
@@ -539,6 +540,26 @@ def __init__(self, rule):
             msg += ' This should be simplified to use only one or the other.'
             logging.warning(msg)
 
+        self.reset_jira_args()
+
+        try:
+            self.client = JIRA(self.server, basic_auth=(self.user, self.password))
+            self.get_priorities()
+            self.get_arbitrary_fields()
+        except JIRAError as e:
+            # JIRAError may contain HTML, pass along only first 1024 chars
+            raise EAException("Error connecting to JIRA: %s" % (str(e)[:1024]))
```

Comment:
Don't do this, it loses the inner stack. Use the overload of raise that takes the `sys.exc_info()[2]`